src/blitter/common.hpp

```diff
@@ -192,4 +192,24 @@ void Blitter::DrawLineGeneric(int x1, int y1, int x2, int y2, int screen_width,
 	}
 }
 
+template <typename T>
+/* static */ void Blitter::MovePixels(const T *src, T *dst, size_t width, size_t height, size_t pitch)
+{
+	if (src == dst) return;
+
+	if (src > dst) {
```

Comment:
Isn't this just the other way around?
```suggestion
	if (src < dst) {
```